config/environments/integration.rb

```diff
@@ -0,0 +1,113 @@
+# Settings for integration testings.  A mix between development and production,
+# using production as a base and pulling in specific options from development
+# where appropriate.
+#
+Vmdb::Application.configure do
+  # Settings specified here will take precedence over those in config/application.rb
+
+  config.eager_load_paths = []
+
+  # Code is not reloaded between requests unless CYPRESS_DEV is set
+  #
+  # Idea borrowed from:
+  #
+  #   https://blog.simplificator.com/2019/10/11/setting-up-cypress-with-rails/
+  #
+  # We want this to be "production-like" unless CYPRESS_DEV is present
+  config.cache_classes = ENV['CYPRESS_DEV'].blank?
+  config.eager_load = false
+
+  # Log error messages when you accidentally call methods on nil.
+  #
+  # Pulled from `config/environment/development.rb`, but only enable if
+  # ENV['CYPRESS_DEV']
+  #
+  config.whiny_nils = true if ENV['CYPRESS_DEV'].present?
+
+  # Full error reports are disabled and caching is turned on
+  #
+  # Same in development and production
+  config.consider_all_requests_local       = false
+
+  # Enable caching by default, but disable it when running with CYPRESS_DEV
+  #
+  # Same default as production, setting CYPRESS_DEV sets it to development-like
+  config.action_controller.perform_caching = ENV['CYPRESS_DEV'].blank?
+
+  # Don't care if the mailer can't send
+  #
+  # Matching development here, commented out on production
+  config.action_mailer.raise_delivery_errors = false
+
+  # Print deprecation notices to the Rails logger.
+  #
+  # Same in development and production
+  config.active_support.deprecation = :log
+
+  # Disable serving static files from the `/public` folder by default since
+  # Apache or NGINX already handles this.
+  #
+  # Using production environment variable, and added headers addition from
+  # test.rb (`.enabled` should default to `true` in `Rails.env.development?`)
+  if ENV['RAILS_SERVE_STATIC_FILES'].present?
```

Comment:
this probably needs to be always enabled for cypress?

(when would apache be running during spec:integration?)

Reply:
@himdel if you are running an appliance (of some sort) in this mode?

I also do "enable" this by default using the `rake` task:

```ruby
  task :env do
    ENV["RAILS_ENV"] = "integration"
    ENV["RAILS_SERVE_STATIC_FILES"] = "true"
  end
```

Reply:
oh ok, as long as it's not an oversight :) :+1:

Reply:
@himdel yeah, admittedly it is probably better viewing this PR through the "lens" of this one:

https://github.com/ManageIQ/manageiq-ui-classic/pull/7194

Which is where I actually make use of it in practice.